public/index.html

```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Author Haven</title>
```

Comment:
I think it will be better to have a dynamic title

Reply:
This title should be handled by everyone in their containers by using document.title in componentDidMount.
`componentDidMount(){
    document.title = "My Page Title"
  }`

Reply:
okay we will all add it, can you do the same here?